src/types/rest/response/account.ts

```diff
@@ -0,0 +1,68 @@
+export interface Position {
+  cost: number;
+  cumulativeBuySize: number;
+  cumulativeSellSize: number;
+  entryPrice: number;
+  estimatedLiquidationPrice: number;
+  future: string;
+  initialMarginRequirement: number;
+  longOrderSize: number;
+  maintenanceMarginRequirement: number;
+  netSize: number;
+  openSize: number;
+  realizedPnl: number;
+  recentAverageOpenPrice: number;
+  recentBreakEvenPrice: number;
+  recentPnl: number;
+  shortOrderSize: number;
+  side: string;
+  size: number;
+  unrealizedPnl: number;
+  collateralUsed: number;
+}
+
+export type Positions = Array<Position>
+export interface AccountSummary {
+  accountIdentifier: number;
+  username: string;
+  collateral: number;
+  freeCollateral: number;
+  totalAccountValue: number;
+  totalPositionSize: number;
+  initialMarginRequirement: number;
+  maintenanceMarginRequirement: number;
+  marginFraction: number;
+  openMarginFraction: number;
+  liquidating: boolean;
+  backstopProvider: boolean;
+  positions: Positions;
+  takerFee: number;
+  makerFee: number;
+  leverage: number;
+  futuresLeverage: number;
+  positionLimit: number;
+  positionLimitUsed: number;
+  useFttCollateral: boolean;
+  chargeInterestOnNegativeUsd: boolean;
+  spotMarginEnabled: boolean;
+  spotLendingEnabled: boolean;
+}
+
+export interface HistoricalBalance {
+  /** @description name of subaccount */
+  account: string
+  /**
+   * @example "BTC-PERP"
+   * @desciprtion	name of product (future, option, token, or currency)
```

Comment:
Great of you to add docs too! Small typo here
```suggestion
   * @description	name of product (future, option, token, or currency)
```